### DISCUSSION TITLE
Inscontient files and attachments kwarg

### DISCUSSION THREAD

**Discussion by JDJGInc** [Rapptz/discord.py#7552] [Q&A] [answered]
Why is message.edit attachments?
and why is await ctx.send files?
shouldn't message.edit be files?

is there a paticular reason for this?

**Comment by JDJGInc** [comment]
is this because edit supports attachments as it works with the attachments already there and new files?

**Comment by LeoCx1000** [comment, marked as answer]
I'd assume so, becase you can now pass `Attachment`s and `File`s.
Also the `attachments` kwarg in message.edit was already a thing before, so it would just be an unnecessary breaking change.
